.pre-commit-config.yaml

```diff
@@ -49,7 +49,6 @@ repos:
     - --output-format=colorized
     - --score=n
     - --enable=local
```

Comment:
Could you move the CLI arguments to the config? This allows other software calling pylint directly, and not through pre-commit, have it run in the same mode.

Reply:
Yes, I will change.

Reply:
Marking as resolved.